components/ModEditSubmissionForm.vue

```diff
@@ -712,6 +737,16 @@ onMounted(() => {
     )
 })
 
+// This doesn't work consistently as sometimes the to.path is not '/moderation', it also blocks our save & exit button and triggers when making this component visable. Also doesn't work at all when imported from vue-router.
```

Comment:
Question: Why do we care about the to/from path here? Is it necessary for the modal to work? Let's say the user accidentally clicks a button that will take them to a different route, shouldn't we still display the modal?

Reply:
At the time of writing that specific code, I cared about the path because it was the only way to get the modal to function as intended. The ticket was to create a modal that pops up when the back button in the browser is clicked, so that's what I focused on. I do agree that focusing on a more broad approach might have been more logical.